### PR TITLE
ops/debug: add more content

### PR DESCRIPTION
**Pull request by kxxt**
- add new strace example
- mention pstack and retsnoop

~~如果可以的话我还想推销一下https://github.com/kxxt/tracexec~~

### CODE REVIEW

File: docs/ops/debug.md [hunk 1]
```diff
@@ -71,6 +71,8 @@ logrotate 会定期（一般是每天，或者文件足够大的时候，请参
 - `strace -p <pid>`：跟踪指定 PID 的进程的系统调用
 - `strace -e openat ls`：只跟踪 `openat` 系统调用
 - `strace -ff -o /tmp/test.log bash`：将 `bash` 及其 fork 出的子进程的系统调用输出到 `/tmp/test.log.*`
+- `strace -k -yy -e mmap ls`: 跟踪 `ls` 的 `mmap` 系统调用，并打印出执行此系统调用时的堆栈（`-k`），同时解码所有文件描述符（`-yy`）。
+这样即可追踪到每个被 `mmap` 的文件在程序的何处被引入
```
Comment (taoky) [inline docs/ops/debug.md:75]:
```suggestion
- `strace -k -yy -e mmap ls`: 跟踪 `ls` 的 `mmap` 系统调用，并打印出执行此系统调用时的堆栈（`-k`），同时解码所有文件描述符（`-yy`）。这样即可追踪到每个被 `mmap` 的文件在程序的何处被引入。
```

**Comment by taoky** [comment]
> ~如果可以的话我还想推销一下https://github.com/kxxt/tracexec~

欢迎！看起来是很有意思的工具。

File: docs/ops/debug.md (hunk 1 above)
Comment (kxxt) [reply docs/ops/debug.md]: Fixed. What's the line length limit BTW?

Comment (taoky) [reply docs/ops/debug.md]: > Fixed. What's the line length limit BTW?

No line length limit. Some contents were written in that style before, but it does not look good for Chinese contents so we would prefer to write a paragraph in one line for now.

Comment (taoky) [reply docs/ops/debug.md]: （还是漏了个句号

Comment (taoky) [reply docs/ops/debug.md]: > > （还是漏了个句号
> 
> 已修复， 不过好像其他列表项并没有句号（

OK，我没有注意到这一点，当时是看这句话太长了，最后缺个标点很奇怪。之后等内容差不多完成了可能会考虑统一一下标点之类的。

**Review by taoky — APPROVED** [review]

**Pull request merged**